test/integration/consul-container/test/gateways/http_route_test.go

```diff
@@ -79,17 +79,22 @@ func TestHTTPRouteFlattening(t *testing.T) {
 	)
 
 	namespace := getNamespace()
+	if namespace != "" {
+		ns := &api.Namespace{Name: namespace}
+		_, _, err := client.Namespaces().Create(ns, nil)
+		assert.NoError(t, err)
```

Comment:
I would use `require` here since the rest of the test is invalid if this assertion doesn't pass

```suggestion
		require.NoError(t, err)
```

Reply:
oh TIL that `assert` doesn't exit the test